app/views/wikis/index.html.erb

```diff
@@ -5,7 +5,13 @@
       <div class="media">
         <div class="media-body">
           <h4 class="media-heading">
-            <%= link_to wiki.title, wiki %>
+            <% if current_user.admin? || current_user.premium? %>
```

Comment:
I see what you are doing here t filter which wikis are displayed. This code looks like it works, but there is a better way. Tomorrow let's discuss converting this to a scope that you can call in the controller.